packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -409,13 +407,13 @@ class EditingParts {
     this.rightSide,
   });
 
-  final String activeWord;
+  final String? activeWord;
 
-  final String leftSide;
+  final String? leftSide;
 
-  final String rightSide;
+  final String? rightSide;
```

Comment:
make all of these non-nullable and required

Reply:
done